DOKS-CI-CD/README.md

```diff
@@ -72,70 +71,72 @@ How this blueprint is structured:
-9. [Knative CLI](https://knative.dev/docs/install/client/install-kn), to interact with `Knative` using the command line interface.
-10. [Kustomize](https://kustomize.io) is extensively used in this guide, and at least some basic knowledge is required. You can follow the [official DigitalOcean tutorial](https://www.digitalocean.com/community/tutorials/how-to-manage-your-kubernetes-configurations-with-kustomize) as a starting point, as well as some real world examples.
+1. A working domain that you own. This is required for exposing public services, such as GitHub webhooks, used in this guide. Make sure to also read the DigitalOcean [DNS Quickstart Guide](https://docs.digitalocean.com/products/networking/dns/quickstart).
+2. A working DOKS cluster running Kubernetes version greater than 1.21. The DOKS cluster must have at least 2 nodes, each with 2 CPUs, 4 GB of memory, and 20 GB of disk storage. For additional instructions on configuring a DOKS cluster, see [How to Set Up a DigitalOcean Managed Kubernetes Cluster (DOKS)](https://github.com/digitalocean/Kubernetes-Starter-Kit-Developers/tree/main/01-setup-DOKS#how-to-set-up-a-digitalocean-managed-kubernetes-cluster-doks).
+3. A [git](https://git-scm.com/downloads) client to interact with GitHub repositories.
+4. [doctl](https://docs.digitalocean.com/reference/doctl/how-to/install) for interacting with DigitalOcean API.
+5. [kubectl](https://kubernetes.io/docs/tasks/tools) for interacting with Kubernetes. Follow these [instructions](https://www.digitalocean.com/docs/kubernetes/how-to/connect-to-cluster/) to connect to your cluster with `kubectl` and `doctl`.
+6. [Helm](https://www.helm.sh) for interacting with Helm releases created by the DigitalOcean 1-Click apps used in this tutorial.
+7. [Argo CLI](https://argo-cd.readthedocs.io/en/stable/cli_installation) to interact with Argo CD using the command line interface.
+8. [Tekton CLI](https://tekton.dev/docs/cli/#installation) to interact with Tekton Pipelines using the command line interface.
+9. [Knative CLI](https://knative.dev/docs/install/client/install-kn) for interacting with Knative using the command line interface.
+10. [Kustomize](https://kustomize.io) is extensively used in this guide, and some basic knowledge is required. You can follow the [official DigitalOcean tutorial](https://www.digitalocean.com/community/tutorials/how-to-manage-your-kubernetes-configurations-with-kustomize) as a starting point.
 
 Next, you will prepare the sample application repository used in this tutorial, as well as the [DigitalOcean Docker Registry](https://www.digitalocean.com/products/container-registry) used for storing application images. You will also create a dedicated Kubernetes namespace, to keep everything clean and well organized.
 
-### Preparing the Sample Application Requirements
+## STEP 1: Prepare the Sample Application Requirements
```

Comment:
I don't understand why is this section prefixed with **STEP 1:**. I think all subsections from here should fall under the **Prerequisites** section. Or, we can move this entire step after the **Getting to Know** sections (implies more work because we need to move around the continuation sentences from each section/subsection as well).

What do you think?

Reply:
Moved the "Getting to Know" sections after "Prerequisites".